operator/cmd/mesh/install.go

```diff
@@ -318,5 +320,5 @@ func getProfileAndEnabledComponents(setOverlay []string, inFilenames []string, f
 			}
 		}
 	}
-	return profile, enabledComponents, nil
+	return profile, iop.Namespace, enabledComponents, nil
```

Comment:
I thought the returned namespace should be the namespace in which istio control plane will be installed, right?

Reply:
@morvencao thanks!  I updated the code to use istioNamespace, let me know if it makes sense.